### DISCUSSION TITLE
Pre-release vs final

### DISCUSSION THREAD

**Discussion by mefistotelis** [swfans/syndwarsfx#150] [General]
A thread intended for comparing screenshots from development vs from final game.

**Bahrain**

![lvl-bahrain-merged1](https://github.com/swfans/swars-re-helpers/assets/11463396/869a1f0c-0888-44b7-8ef9-aae8379e78d5)
![lvl-bahrain-merged1-in-final](https://github.com/swfans/swars-re-helpers/assets/11463396/ee32c980-24b5-40fd-889f-5fb7fa69fec6)

* It is not visible how exactly the old C-shaped building looks like, but it is different.
* Statue within the C is different as well.
* The differences in lighting are noticeable.
* Laser shoots in pulses. Though I didn't managed to capture final laser shot on the re-reation.
* HUD is from before medkit button was added.
* Floor tiles are rough, like in pre-alpha.

**Comment by mefistotelis** [comment]
**Tokyo**

Top right corner of the map:
![lvl-tokyo-pre-pc_games-n045-1996_06-pg0370-ss02](https://github.com/swfans/swars/assets/11463396/26a302c2-d409-486c-b028-44e8fc781283)
![lvl-tokyo-pre-pc_games-n045-1996_06-pg0370-ss02-in-final](https://github.com/swfans/swars/assets/11463396/2d6e902a-3c83-4fc4-a21c-b34f414354ee)

* Bezier road leading up was removed, domes and windmills added in its place.
* Advert screen removed.
* Buildings textured differently.
* Pedestrian crossings placed differently.
* Interface from before medikit icon.
* No building within the walled area.

**Comment by Moburma** [comment]
> * Laser shoots in pulses. Though I didn't managed to capture final laser shot on the re-reation.

I think it just worked as in the pre-alpha demo. That is it fired an instantaneous beam like the final game, but then it decays into these little mini laser pulses. You can see it here at about 10:35: 
https://youtu.be/GItY8-izShw?t=639
Based on screenshots I think they kept this effect until around the Beta era of development. I think it's in the trailer video as well, but it's hard to see as a) it's sped up footage and b) the characters are close up to each other

**Comment by mefistotelis** [comment]
>  it fired an instantaneous beam like the final game, but then it decays into these little mini laser pulses.

Right. I somehow never registered that.

**Tokyo**

Here's the same area as previously, but rotated:

![lvl-tokyo-merged1](https://github.com/swfans/swars/assets/11463396/924855da-3bd2-4bc5-ac8d-faa76425ee09)
![lvl-tokyo-merged1-in-final](https://github.com/swfans/swars/assets/11463396/77f3bd72-9517-4adc-b3a4-58194b57e43a)

* There is a light grey building which was removed in final.
* Behind the building, there's another thing.. not sure what. Or maybe it's just a print artifact, as the minimap clearly shows only one building.
* Visible part of a sphinx/bullfrog in place of a stationary laser.
* Minimap shows that the bezier road leads to an elevated area. Must be a very small one - that's near border of the map.

**Reply by Moburma** [reply]
The road here is really intriguing, I wonder if it was like the early screenshots of the Cairo map in the press kit where it shows the map originally extended further along but was totally cut off in the final version. Also the video screen is showing the cut Creation advert that was only in the beta demo, so it at least confirms that really was intended to be in the game at one point.

**Reply by mefistotelis** [reply]
I don't have that press kit, can you share it? I have Cairo full map screenshot from an advert in "pc gamer usa 1997.02", but that one doesn't look much different than final. Well I'm not able to tell if it has the IML rail, as it's too dark and low resolution.

Also bigger maps would be surprising as the game uses 127x127 tile maps, with computations often on signed values. So some functions would treat tile 128 as negative, while others would process it properly. The sign issue was solved in DK, where they used 85 large tiles consisting of 3 sub-tiles, so map size in DK is 255x255 sub-tiles.

**Reply by Moburma** [reply]
I don't think the map was bigger, they just moved the contents around. I wonder if either they moved the elements slightly, or if it was simply shifted in the "frame" so that there was more space on the opposite side of the level. The examples I'm talking about can be seen here:

https://tcrf.net/Prerelease:Syndicate_Wars_(DOS)#Early_Map_Overviews

Someone uploaded this ages ago, a zip that has the Forthcoming Products from Bullfrog demo as well as a collection of press packs for their games:

https://www.dosgamesarchive.com/file/syndicate-wars/bfrgprev/

I'm not sure if it was found like this on a magazine, or they just collected them together that way (think the latter). They're probably off Score magazine otherwise, they seemed to love putting this stuff on their CD (most UK mags didn't like doing this, likely as it exposes the fact they were just copying the generic press releases into articles rather than actually reporting on an "exclusive").

**Reply by mefistotelis** [reply]
Oh, I see what's going on.

Yes, I have all these shots, and they are from CDs, probably Score/Level.

So, please take a detailed look at the last picture from "high res" chapter you've uploaded there, the one from Tokyo:
https://tcrf.net/File:SynWars-SYNDIC09.png

Notice the elevated tree in red pot on top, and which types of buildings are on the left. Also the terrain curve up on the right.

Now compare that to what you've identified as Cairo ;).

**Reply by Moburma** [reply]
Ahh, well spotted. To be fair it made more sense when comparing strictly against the final game maps. Weird they changed that bit when they clearly thought it interesting enough to take not one but two pictures of its layout.

**Comment by mefistotelis** [comment]
**Tokyo**

This time a different area, back of the bank and the central hill:

![lvl-tokyo-pre-the_games_machine-n083_ita-1996_02-pg037-ss01](https://github.com/swfans/swars/assets/11463396/ad9a65d3-0a25-4d9b-b342-762659eeb0c9)
![lvl-tokyo-pre-the_games_machine-n083_ita-1996_02-pg037-ss01-in-final](https://github.com/swfans/swars/assets/11463396/7ec16fc9-8d90-4b38-84a3-5eb6735a6bd9)

* Different types of buildings on the hill.
* Different floor tiles in some places.
* Bank on a bit different angle.
* Different sprites for trees.

**Reply by mefistotelis** [reply]
**Rome**

Area near start of the mission, where the one-floor building explodes.

![lvl-rome-pre-pc_zone-10_ita-pg031-ss02](https://github.com/swfans/swars/assets/11463396/ae8e648c-5f1a-4d85-a9d4-3de48ed43cec)
![lvl-rome-pre-pc_zone-10_ita-pg031-ss02-in-final](https://github.com/swfans/swars/assets/11463396/f342d8c4-f8f1-410c-b6bd-246a576b065d)

* There is a second building, two floor one.

**Comment by mefistotelis** [comment]
**Buenos Aires**

In front of the bank there is a parking, and at end of that parking:

![lvl-buenos_aires-merge1](https://github.com/swfans/swars/assets/11463396/adbdce91-2d43-465c-ba41-2d951771c322)
![lvl-buenos_aires-merge1-in-final](https://github.com/swfans/swars/assets/11463396/25d31067-9660-4297-9568-f6df0ca5e264)

* There was no cemetery originally, the huge two-part building was there instead.
* There was one more building with X-shape minimap footprint.
* The parking area was a bit larger.

**Comment by mefistotelis** [comment]
**Phoenix**

This map was prominent in pre-release screenshots, but it didn't changed much.

Like here, in the triple X buildings shot:
![lvl-phoenix-pre-pc_spiel-1996-04-pg025-ss01](https://github.com/swfans/swars/assets/11463396/5b8afe43-b627-4efa-99fb-ada1dca061ec)
![lvl-phoenix-pre-pc_spiel-1996-04-pg025-ss01-in-final](https://github.com/swfans/swars/assets/11463396/cca0098f-b826-488f-9147-a35e4ca95974)

* Textures switched for the buildings.
* Minimap suggests no other changes nearby.

**Comment by mefistotelis** [comment]
**London**

The podium area:
![lvl-london-pre-computer-and_video_games-no178-pg047-ss05](https://github.com/swfans/swars/assets/11463396/420ec3e6-6ecd-4209-bd2a-7aac4db67f70)
![lvl-london-pre-computer-and_video_games-no178-pg047-ss05-in-final-orig](https://github.com/swfans/swars/assets/11463396/bfd1a7cb-5c45-4b67-b107-d69fbd58804a)

* No podium in early version.
* Different style of the billboard screen frame.
* The building behind the billboard screen switched to different model.
* HUD sprites not yet scaled to high resolution

**Comment by mefistotelis** [comment]
Cape Town

The bottom part viewed from a far:
![lvl-cape_town-SYNII032](https://github.com/swfans/swars/assets/11463396/ec160b17-a510-4299-aea2-e64f92b51dbb)
![lvl-cape_town-SYNII032-in-final](https://github.com/swfans/swars/assets/11463396/7980437b-a647-4034-a69b-51e42914eb89)

* Simple building in place of Zealot Church.
* Different statue in front of the Church.
* Differences in textures of several buildings.
* Buildings cast shadows on one side, like there was a sun

**Comment by mefistotelis** [comment]
**Colombo**

Whole map (almost):
![lvl-colombo-SYNII036](https://github.com/swfans/swars/assets/11463396/11f65c88-b51e-4ec2-8106-6d353033c797)
![lvl-colombo-SYNII0360in-final](https://github.com/swfans/swars/assets/11463396/8a5fca6d-173c-473a-a8f8-9ec48c86e50d)

* Tall quad buildings have different textures.
* Orbital elevator was added.
* Satellite missile silos were added.
* The mecha walkers were removed.
* Military vehicles were removed, replaced by Zealot vehicles, and more of them.
* Lights were simplified, increasing global illumination and removing lights in cavities within the central hill.

**Reply by Moburma** [reply]
This one brings home how huge those military mechs were going to be - they're probably the biggest vehicles in the game. It's not clear if they were actually going to be moveable vehicles or just static background objects, but they were massive.

**Reply by mefistotelis** [reply]
Yes, even as static objects seeing these mechas would be cool. These remind me of the beginning of Stanislaw Lem's book "Fiasco", where the main character dies for the first time inside such walker.

Hm, thinking about it, this book also ends with messing with a planets ionosphere.

**Comment by mefistotelis** [comment]
**New Delhi**

Port island with docks:

![lvl-new_delhi_SYNDIC05](https://github.com/swfans/swars/assets/11463396/af9d4f3c-e34e-44b5-9b2f-0647bc081967)
![lvl-new_delhi_SYNDIC05-in-final](https://github.com/swfans/swars/assets/11463396/57578ada-4cab-4ed9-9123-77404ec4cf7f)

* Different ground textures - helipad marking removed, instead a darker path added between buildings.
* Different boats - one modified, one completely different.
* Different buildings with different textures.
* Wobbly texture left at map border in the final.

These present the same concept, and the same ground shape, but are so different that I wonder whether it's really the same map, or one re-created from scratch.
